packages/git/src/browser/git-quick-open-service.ts

```diff
@@ -47,7 +49,8 @@ export class GitQuickOpenService {
         @inject(QuickOpenService) protected readonly quickOpenService: QuickOpenService,
         @inject(MessageService) protected readonly messageService: MessageService,
         @inject(WorkspaceService) protected readonly workspaceService: WorkspaceService,
-        @inject(FileSystem) protected readonly fileSystem: FileSystem
+        @inject(FileSystem) protected readonly fileSystem: FileSystem,
+        @inject(LabelProvider) protected readonly labelProvider: LabelProvider
```

Comment:
Can we refactor `labelProvider` as a property injection over a constructor injection.
Injecting new constructor injections is a breaking change based on our [coding guidelines](https://github.com/theia-ide/theia/wiki/Coding-Guidelines#property-injection)